CHANGELOG.md

```diff
@@ -2,6 +2,24 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
+## Mapbox Navigation SDK 2.0.0-beta.9.5 - Aug 24, 2021
+
```

Comment:
```suggestion
This is a patch release on top of `v2.0.0-beta.9.x` which does not include changes introduced in `v2.0.0-beta.10` and later. The `v2.0.0-beta.23` remains the latest and most up-to-date release.

```

We've run into a little bit of confusion in the past about these patch releases.

Reply:
This won't be needed for cherry-pick, correct?

Reply:
Not necessarily, depending on how you'll update the changelog in the other branch. If you'll synchronize the changelogs, then best to leave the note.